### PR TITLE
Login Branch from Registration Branch

### PR DESCRIPTION
**Pull request by cindyzhang12**
## Summary

- this branch branches off the registration branch since it is necessary to create users for login

### CODE REVIEW

File: src/components/pages/ConfirmOTP/ConfirmOTP.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    props.location ? props.location.state.authCode : undefined
+  );
+
+  const handleuserAuthCodeChange = () => (event: React.ChangeEvent<HTMLInputElement>) => {
+    setuserAuthCode(event.target.value);
+  };
+
+  const handleSubmit = () => {
+    if (userAuthCode == correctAuthCode) {
+      window.location.href = REACT_APP_GLIFIC_CHAT_REDIRECT;
```
Comment (cindyzhang12) [inline src/components/pages/ConfirmOTP/ConfirmOTP.tsx:27]: Is this method of redirecting the user to the chat page okay? Is there a better way to do it?

Comment (kurund) [reply src/components/pages/ConfirmOTP/ConfirmOTP.tsx:27]: You can use <Redirect> to send to chat page.

File: src/config/axios.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export const REACT_APP_GLIFIC_REGISTRATION_API =
+  process.env.REACT_APP_GLIFIC_API + '/v1/registration';
+export const REACT_APP_GLIFIC_AUTHENTICATION_API =
+  process.env.REACT_APP_GLIFIC_API + '/v1/registration/send_otp';
+export const REACT_APP_GLIFIC_CHAT_REDIRECT = 'http://localhost:3000/chat';
```
Comment (kurund) [inline src/config/axios.ts:5]: We don't need this.

File: src/common/constants.ts [hunk 3]
```diff
@@ -1,3 +1,10 @@
 export const SIDE_DRAWER_WIDTH = 200;
 export const DATE_FORMAT = 'DD/MM/YY';
 export const TIME_FORMAT = 'HH:mm';
+export const REACT_APP_GLIFIC_REGISTRATION_API =
+  process.env.REACT_APP_GLIFIC_API + '/v1/registration';
+export const REACT_APP_GLIFIC_AUTHENTICATION_API =
+  process.env.REACT_APP_GLIFIC_API + '/v1/registration/send_otp';
+export const REACT_APP_GLIFIC_CHAT_REDIRECT = 'http://localhost:3000/chat';
```
Comment (kurund) [inline src/common/constants.ts:8]: We should not hardcode this here. I don't think we need redirect URL. We can directly send the user to the route.

**Comment by kurund** [comment]
@cindyzhang12 Can you please check the comments on https://github.com/glific/glific-frontend/pull/114 and just add it in this branch.

**Pull request merged**